docs/reference/datatiers.asciidoc

```diff
@@ -2,40 +2,66 @@
 [[data-tiers]]
 == Data tiers
 
-A _data tier_ is a collection of nodes with the same data role that
-typically share the same hardware profile:
-
-* <<content-tier, Content tier>> nodes handle the indexing and query load for content such as a product catalog.
-* <<hot-tier, Hot tier>> nodes handle the indexing load for time series data such as logs or metrics
-and hold your most recent, most-frequently-accessed data.
-* <<warm-tier, Warm tier>> nodes hold time series data that is accessed less-frequently
+A _data tier_ is a collection of <<modules-node,nodes>> within a cluster which share the same 
+<<node-roles,data node role>>. Elastic recommends this collection of nodes also shares the same 
+hardware profile to avoid <<hotspotting,hot spotting>>. Data tiers' usage generally splits along 
+<<data-management,data categories>> for _content_ and _time series_ data. {es} available 
+data tiers:
```

Comment:
IMO (said strongly, held loosely) the important flag is excellent as a stand-alone call out; however, I feel including the reason to Elastic's recommendation within its paragraph is in itself helpful/important. 

Secondly and separately, I believe it's important to clarify what "content" and "time series" data is before using those terms within the next sections bullet list, since those are not industry terms but are lower-case proper nouns we introduce.